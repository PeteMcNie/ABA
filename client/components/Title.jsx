import React from 'react'
import { Navbar, Nav, Button } from 'react-bootstrap'

import RegisterModal from './RegisterModal'
import SignInModal from './SignInModal'

// import Logout from './Logout'

function Title () {
  const [modalShow, setModalShow] = React.useState(false)

  return (
    <>
      <Navbar bg="light" variant="light">
        <Navbar.Brand href="#home">Balance</Navbar.Brand>
        <Nav className="mr-auto">
          {/* <Nav.Link href="#home">Home</Nav.Link>
        <Nav.Link href="#features">Features</Nav.Link>
        <Nav.Link href="#pricing">Pricing</Nav.Link> */}
        </Nav>
        <Button variant="secondary" size="lg mr-2" onClick={() => setModalShow(true)}>Register</Button>
        <Button variant="secondary" size="lg">Sign In</Button>

        {/* <SignIn /> */}
        {/* <Logout /> */}
      </Navbar>
      <RegisterModal show={modalShow} onHide={() => setModalShow(false)}/>
      <SignInModal show={modalShow} onHide={() => setModalShow(false)}/>
    </>
  )
}

export default Title
