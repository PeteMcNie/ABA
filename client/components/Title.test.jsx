import React from 'react'

import { renderWithRedux } from '../testing/utils'

import Title from './Title'

test('Component to match snapshot', () => {
  const { container } = renderWithRedux(<Title />)
  expect(container).toMatchSnapshot()
})
