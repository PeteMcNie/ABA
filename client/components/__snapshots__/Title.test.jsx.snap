// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component to match snapshot 1`] = `
<div>
  <nav
    class="navbar navbar-expand navbar-light bg-light"
    id="title"
  >
    <span
      class="navbar-brand"
    >
      Balance
    </span>
    <div
      class="mr-auto navbar-nav"
    />
    <button
      class="btn btn-secondary btn-lg mr-2"
      type="button"
    >
      Register
    </button>
    <button
      class="btn btn-secondary btn-lg mr-2"
      type="button"
    >
      Sign In
    </button>
  </nav>
</div>
`;
